packages/quick_actions/quick_actions_ios/ios/Classes/quick_actions_ios-umbrella.h

```diff
@@ -7,4 +7,4 @@
 FOUNDATION_EXPORT double quickActionsIOSVersionNumber;
 FOUNDATION_EXPORT const unsigned char quickActionsIOSVersionString[];
 
-#import <quick_actions_ios/FLTQuickActionsPlugin.h>
+#import <quick_actions_ios/FLTShortcutStateManager.h>
```

Comment:
Since the custom module map is gone, this has to be here, until the migration is done (refer to the description of this PR)